### PR TITLE
Add standalone `mock-tee-primitives`

### PR DESCRIPTION
**Pull request by Kailai-Wang**
This is to break the cyclic dependency of the `parachain` repo and the `tee-worker` repo.

This primitive should only be used by mock pallet.

In this way we make sure we don't have blockers when updating the polkadot dependencies.

### CODE REVIEW

**Review by BillyWooo — APPROVED** [review]
LGTM

File: mock-tee-primitives/src/identity.rs [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	Substrate(SubstrateNetwork),
+	Evm(EvmNetwork),
+}
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum Web2Network {
+	Twitter,
+	Discord,
+	Github,
```
Comment (montogeek) [inline mock-tee-primitives/src/identity.rs:65]: @Kailai-Wang Are you using brand names here or just the name? 
Correct brand name for "Github" is "GitHub".

Comment (Kailai-Wang) [reply mock-tee-primitives/src/identity.rs:65]: > @Kailai-Wang Are you using brand names here or just the name? Correct brand name for "Github" is "GitHub".

It's just a name string for GitHub.
Does it make F/E's life easier if we change to `GitHub`?

Comment (montogeek) [reply mock-tee-primitives/src/identity.rs:65]: No, it is just me trying to use the correct name, thanks!

**Pull request merged**